### ISSUE TITLE
v0.28.1-0 QA Review

### ISSUE THREAD

**Issue by smashwilson** [atom/github#2130]
#### Changes to verify

- [x] #2100: Handle ghost users (deleted accounts)
- [x] #2104: Get author info tooltips working again
- [x] #2105: Don't pass destroyed TextEditors to find-and-replace
- [x] #2066: Hide alt text when committer's avatar fails to load
- [x] #2057: Indicate that comment has been edited
